src/videojs.ga.coffee

```diff
@@ -53,7 +55,7 @@ videojs.plugin 'ga', (options = {}) ->
         if "start" in eventsToTrack && percent == 0 && percentPlayed > 0
           sendbeacon( 'start', true )
         else if "percentsPlayed" in eventsToTrack && percentPlayed != 0
-          sendbeacon( 'percent played', true, percent )
+          sendbeacon( 'percent played'+(percent-percentsPlayedInterval)+"-"+percent, true, 1)
```

Comment:
hello true regarding the possibility of breaking the plugin but in fack before we were unable to understand who read what.
Possibility is to create an other tag and send also the old one. I'm not fan of that.

This patch make sense for me